server/src/test/java/io/unitycatalog/server/base/BaseServerTest.java

```diff
@@ -10,19 +16,31 @@
 import org.hibernate.Transaction;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.mockito.Mock;
 
 public abstract class BaseServerTest {
 
   public static ServerConfig serverConfig = new ServerConfig("http://localhost", "");
   protected static UnityCatalogServer unityCatalogServer;
   protected static Properties serverProperties;
   protected static HibernateConfigurator hibernateConfigurator;
+  protected static CredentialOperations credentialOperations;
+
+  @Mock AwsCredentialVendor awsCredentialVendor;
+  @Mock AzureCredentialVendor azureCredentialVendor;
+  @Mock GcpCredentialVendor gcpCredentialVendor;
 
   protected void setUpProperties() {
     serverProperties = new Properties();
     serverProperties.setProperty("server.env", "test");
   }
 
+  protected void setUpCredentialOperations() {
+    awsCredentialVendor = mock(AwsCredentialVendor.class);
+    azureCredentialVendor = mock(AzureCredentialVendor.class);
+    gcpCredentialVendor = mock(GcpCredentialVendor.class);
```

Comment:
whats the point of defining these in the base class if the mocks are not even setup? 
I am guessing that any test that needs to work with mocked cred vendors will extend BaseCRUDTestWithMockCreds. So why not just move this entire thing completely in that class?